pallets/portfolio/src/lib.rs

```diff
@@ -237,47 +219,9 @@ decl_module! {
             Self::deposit_event(Event::PortfolioDeleted(primary_did, num));
         }
 
-        /// Moves a token amount from one portfolio of an identity to another portfolio of the same
-        /// identity. Must be called by the custodian of the sender.
-        /// Funds from deleted portfolios can also be recovered via this method.
-        ///
-        /// A short memo can be added to to each token amount moved.
-        ///
-        /// # Errors
-        /// * `PortfolioDoesNotExist` if one or both of the portfolios reference an invalid portfolio.
-        /// * `destination_is_same_portfolio` if both sender and receiver portfolio are the same
-        /// * `DifferentIdentityPortfolios` if the sender and receiver portfolios belong to different identities
-        /// * `UnauthorizedCustodian` if the caller is not the custodian of the from portfolio
-        /// * `InsufficientPortfolioBalance` if the sender does not have enough free balance
-        /// * `NoDuplicateAssetsAllowed` the same ticker can't be repeated in the items vector.
-        ///
-        /// # Permissions
-        /// * Portfolio
-        #[weight = <T as Config>::WeightInfo::move_portfolio_funds(items.len() as u32)]
-        pub fn move_portfolio_funds(
-            origin,
-            from: PortfolioId,
-            to: PortfolioId,
-            items: Vec<MovePortfolioItem>,
-        ) {
-            let primary_did =
-                Self::ensure_portfolios_validity_and_permissions(origin, from.clone(), to.clone())?;
-
-            Self::ensure_valid_balances(&from, &items)?;
-
-            // Commit changes.
-            for item in items {
-                Self::unchecked_transfer_portfolio_balance(&from, &to, &item.ticker, item.amount);
-                Self::deposit_event(Event::MovedBetweenPortfolios(
-                    primary_did,
-                    from,
-                    to,
-                    item.ticker,
-                    item.amount,
-                    item.memo
-                ));
-            }
-        }
+        /// Placeholder for removed `move_portfolio_funds`
+        #[weight = 1_000]
+        pub fn place_holder_move_portfolio_funds(_origin) {}
```

Comment:
We don't need to have place holders in this pallet, there isn't any scheduled tasks here.